<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MrVizifyMain</class>
 <widget class="QMainWindow" name="MrVizifyMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MrVizifyMain</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0" columnstretch="1,0,2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="gBox_LoadImage">
      <property name="title">
       <string>Load Images</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>111</width>
         <height>71</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="vBox_LoadImage">
        <item>
         <widget class="QPushButton" name="pBtn_LoadDicom">
          <property name="text">
           <string>Load DICOM</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pBtn_LoadData">
          <property name="text">
           <string>Load Data</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="gBox_Algs">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>261</width>
         <height>131</height>
        </rect>
       </property>
       <property name="title">
        <string>Processing Modules</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>241</width>
          <height>101</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gLayout_modules">
         <item row="0" column="0">
          <widget class="QPushButton" name="pBtn_Threshold">
           <property name="text">
            <string>Threshold</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Statistics</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pushButton_6">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="0" column="1" rowspan="3">
     <widget class="Line" name="hLine">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="3">
     <layout class="QVBoxLayout" name="vBox_Viz" stretch="0,1">
      <item>
       <layout class="QHBoxLayout" name="hBox_Vizbtn" stretch="0,1">
        <item>
         <widget class="QPushButton" name="pBtn_ViewImage">
          <property name="text">
           <string>View</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cBox_Volumes"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QVTKOpenGLNativeWidget" name="wGet_ViewWindow" native="true"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="hBox_dicomLabel">
      <item>
       <widget class="QLabel" name="label_Dicom">
        <property name="text">
         <string>Dicom:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lEdit_Dicom">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pBtn_VizAll">
      <property name="text">
       <string>Visualize All Images</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLNativeWidget</class>
   <extends>QWidget</extends>
   <header>QVTKOpenGLNativeWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
